cmd/wallet-web/src/components/VaultCard/VaultCard.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div>
+    <div
+      v-if="type === 'regular'"
+      class="
+        md:flex-col
+        px-6
+        w-full
+        md:w-64
+        h-20
+        md:h-auto
+        bg-neutrals-white
+        rounded-xl
+        flex flex-row
+      "
+    >
+      <div class="pt-5">
+        <div
+          :class="[`flex justify-center items-center w-12 h-12 bg-gradient-${color} rounded-full`]"
+        >
+          <img class="w-6 h-5" src="@/assets/img/vaults.svg" alt="Vault Icon" />
+        </div>
+      </div>
+      <div>
+        <div class="px-3 md:px-0 pb-4">
+          <span class="block pt-4 text-lg font-bold text-neutrals-dark"> {{ text }}</span>
+          <span class="block text-sm text-neutrals-medium">
+            {{ credentialFound }}
+          </span>
+        </div>
+      </div>
+    </div>
+    <div
+      v-if="type === 'addVault'"
```

Comment:
Maybe we could make it `addNew`? Since we pass this prop to the `VaultCard` component, it is already clear that this has to do with the vault